riotfile.py

```diff
@@ -577,6 +577,8 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             command="python tests/wait-for-services.py {cmdargs}",
             # Default Python 3 (3.10) collections package breaks with kombu/vertica, so specify Python 3.9 instead.
             pys="3.9",
+            create=True,
+            skip_dev_install=True,
```

Comment:
Is this change still needed after the CircleCI changes? I'd expect `-s` to do the job here if we have valid base venvs

Reply:
I seriously had mixed results with `create` and `skip_dev_install`... better safe than sorry?

Reply:
but that was also before I realized we weren't restoring the original base venvs properly.

I think I noticed that with `create` if the base venv didn't exist, it would still try to create it and install the dev package even if both of these were set.